app/src/main/java/org/simple/clinic/home/HomeScreenUiRenderer.kt

```diff
@@ -8,6 +8,18 @@ class HomeScreenUiRenderer(private val ui: HomeScreenUi) : ViewRenderer<HomeScre
     if (model.hasFacility) {
       renderFacility(model)
     }
+
+    if (model.hasOverdueAppointmentCount) {
+      renderOverdueAppointmentCount(model)
+    }
+  }
+
+  private fun renderOverdueAppointmentCount(model: HomeScreenModel) {
+    val overdueAppointmentCount = model.overdueAppointmentCount!!
+    if (overdueAppointmentCount > 0)
```

Comment:
What do you think about making this check a part of `HomeScreenModel` as a computed property, say, `hasAtLeastOneOverdueAppointment`?

Reply:
Oh yeah, I should have done that. Completely missed it, will do it in final PR 👍🏾